src/AppInstallerCommonCore/Public/AppInstallerRuntime.h

```diff
@@ -47,6 +47,8 @@ namespace AppInstaller::Runtime
         PortableLinksMachineLocation,
         // The root location for the package containing the winget application.
         SelfPackageRoot,
+        // The location where user downloads are stored.
+        Downloads,

```

Comment:
Would probably be good to also add this to `OutputKeyDirectories` in RootCommand.cpp